Pages/Search.cshtml

```diff
@@ -27,17 +27,17 @@
                                 <label for="breweryType" class="form-label">Brewery Type</label>
                                 <select class="form-select" id="breweryType" name="BreweryType">
                                     <option value="">Select type</option>
-                                    <option value="micro" selected="@(Model.BreweryType == "micro")">Micro</option>
-                                    <option value="nano" selected="@(Model.BreweryType == "nano")">Nano</option>
-                                    <option value="regional" selected="@(Model.BreweryType == "regional")">Regional</option>
-                                    <option value="brewpub" selected="@(Model.BreweryType == "brewpub")">Brewpub</option>
-                                    <option value="large" selected="@(Model.BreweryType == "large")">Large</option>
-                                    <option value="planning" selected="@(Model.BreweryType == "planning")">Planning</option>
-                                    <option value="bar" selected="@(Model.BreweryType == "bar")">Bar</option>
-                                    <option value="contract" selected="@(Model.BreweryType == "contract")">Contract</option>
-                                    <option value="proprietor" selected="@(Model.BreweryType == "proprietor")">Proprietor</option>
-                                    <option value="closed" selected="@(Model.BreweryType == "closed")">Closed</option>
+                                    @foreach (var type in new[] { "micro", "nano", "regional", "brewpub", "large", "planning", "bar", "contract", "proprietor", "closed" })
```

Comment:
Interesting way to do this, though a good concept.  I'd go one step further and exterlanize this into a separate collection in the codebehind (.cshtml.cs file).  Or, go even further than that and make it configurable.

Either way, it does reduce several lines of code, and is set up for further refactoring (the steps I mention above), so it's a good step towards tech debt reduction.